tools/free_energy/alchemical_run/gmx_fep.xml

```diff
@@ -0,0 +1,518 @@
+
+#if $constraints.cons == "no":
+constraints              = none
+#end if
+
+#if $constraints.cons == "yes":
+constraints              = $constraints.cons_type  
+constraint-algorithm     = lincs
+lincs-order              = $constraints.lincs_order
+lincs-iter               = $constraints.lincs_iter
+lincs-warnangle          = $constraints.lincs_warnangle
+#end if
+
+comm-mode                = Linear
+
+cutoff-scheme            = Verlet
+nstlist                  = 10
+ns_type                  = grid
+pbc                      = xyz
+rlist                    = 0.8
+
+coulombtype              = PME
+coulomb-modifier         = none
+rcoulomb                 = 0.8
+fourierspacing           = 0.10
+pme_order                = 4
+ewald_rtol               = 1.0E-5
+
+vdwtype                  = cut-off
+vdw-modifier             = none
+rvdw                     = 0.8
+DispCorr                 = AllEnerPres
+
+pcoupl                   = Parrinello-Rahman
+pcoupltype               = isotropic
+tau_p                    = 2
+compressibility          = 4.5e-5
+ref_p                    = $pressure
+refcoord-scaling         = com
+
+gen-vel                  = no
+continuation             = yes
+
+ld-seed                  = -1
+gen-seed                 = $seed 
+
+free-energy              = yes
+delta-lambda             = 0  ; do not use slow growth method
+init-lambda-state        = %L%
+
+#if $path.select_path == "global":
+fep-lambdas              = $path.fep
+#end if    
+
+#if $path.select_path in ["default1", "default2"]:
+coul-lambdas             = $path.coul
+vdw-lambdas              = $path.vdw
+bonded-lambdas           = $path.bonded
+#end if
+
+nstdhdl                  = 10
+nstcalcenergy            = 10
+calc-lambda-neighbors    = -1 
+ 
+#if $feoptions.feop == "default":
+sc-alpha                 = 0.5 
+sc-coul                  = no  
+sc-power                 = 1   
+sc-r-power               = 6
+sc-sigma                 = 0.3
+couple-lambda0           = vdw-q
+couple-lambda1           = none
+couple-intramol          = no
+#end if
+
+#if $feoptions.feop == "modify":
+sc-alpha                 = $feoptions.scalpha 
+sc-coul                  = $feoptions.sccoul  
+sc-power                 = $feoptions.scpower    
+sc-r-power               = $feoptions.scrpower
+sc-sigma                 = $feoptions.scsigma
+couple-lambda0           = $feoptions.couplelambda0
+couple-lambda1           = $feoptions.couplelambda1
+couple-intramol          = $feoptions.coupleintramol 
+#end if
+
+dhdl-derivatives         = yes
+dhdl-print-energy        = no
+separate-dhdl-file       = yes
+dh_hist_size             = 0
+dh_hist_spacing          = 0.1
+
+         </configfile>
+    </configfiles>  
+    <inputs>
+        <param format="gro" name="groin" type="data" label="Structure (GRO) file."/>
+        <param format="top" name="topin" type="data" label="Topology (TOP) file."/>
+        <param name="minstep"  type="integer" value="10000" label="Minimization steps"/>
```

Comment:
can you add some help fields to the parameters please, this is really necessary for MD tools IMO

Reply:
ping @tsenapathi